puppet/modules/freight/templates/deploy_debs.erb

```diff
@@ -43,6 +47,17 @@ files.each do |file|
 end
 system "/usr/bin/freight-cache -v -c /home/freight/freight.conf apt/#{os}"
 
+foreman_release_dir = "#{ENV['TARGETCACHE']}/pool/#{os}/#{version}/f/foreman-release/"
+release_files = Dir.glob("#{foreman_release_dir}/foreman-release_*.deb")
+# we assume the file names can be alphabetically sorted to find the newest
```

Comment:
That is assuming we never hit a version 10, right?

Reply:
debian revision 10? yeah, that too, but also https://github.com/theforeman/foreman-infra/pull/1768#discussion_r943273530

Reply:
now sorting by mtime, this should be safe enough.